### ISSUE TITLE
As a user who uses a screen reader, I want to know which page I am on.

### ISSUE THREAD

**Issue by shubhi-raft** [raft-tech/TANF-app#283]
**Description:**
Currently, the authenticated page for a new user has the same page title as the pre-login landing page. Each page should have unique and descriptive site title for screen reader. This issue will deliver updating h1 content prepended to the site name -  “Request Access | TDP - TANF Data Portal ” for the authenticated new user profile page.

**Acceptance Criteria:** 
- [x] Each pages site title starts with the h1 content of the page

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [x] See if using `useEffect` will work for updating page titles
- [x] If it will not work then bring in `react-helmet` package
- [x] Ensure edit profile page title is `Request Access - TDP - TANF Data Portal`
- [x] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
- https://reactjs.org/docs/hooks-effect.html
- https://www.npmjs.com/package/react-helmet

Note:
- May be brought onto Sprint 6 if other frontend isssues have been completed.

**Comment by cmajel** [comment]
adding note from sprint planning's zoom chat from @rahearn 

> On my way out, I'll let you know that I just tested a change that used "useEffect" to update the page title. It definitely works

**Comment by lfrohlich** [comment]
This was merged in PR #94 in Sprint 9 on 12/7/20
